bots/tep-automation/pkg/tep/types.go

```diff
@@ -0,0 +1,147 @@
+package tep
+
+import (
+	"bytes"
+	"fmt"
+	"text/template"
+	"time"
+)
+
+const (
+	// ActionParamName is the param name that will hold the pull request webhook action
+	ActionParamName = "pullRequestAction"
+	// PRNumberParamName is the param name that will hold the pull request number
+	PRNumberParamName = "pullRequestNumber"
+	// PRTitleParamName is the param name that will hold the pull request title
+	PRTitleParamName = "pullRequestTitle"
+	// PRBodyParamName is the param name that will hold the pull request body
+	PRBodyParamName = "pullRequestBody"
+	// PackageParamName is the param name that will hold the pull request's repository owner/name
+	PackageParamName = "package"
+	// PRIsMergedParamName is the param name that will hold whether the pull request is merged
+	PRIsMergedParamName = "pullRequestIsMerged"
+	// GitRevisionParamName is the param name that will hold the HEAD git SHA for the PR
+	GitRevisionParamName = "gitRevision"
+
+	// CommunityBlobBaseURL is the base URL for links to blobs in github.com/tektoncd/community/teps
+	CommunityBlobBaseURL = "https://github.com/tektoncd/community/blob/main/teps/"
+
+	// TrackingIssueBodyTmpl is the template (using text/template) for the body of tracking issues.
+	TrackingIssueBodyTmpl = `This issue tracks TEP-{{ .issue.TEPID }}.
```

Comment:
This is a first stab at the tracking issue body template. @jerop has created https://github.com/tektoncd/community/issues/601 as an attempt to manually do what the bot will end up doing, and I should rework the body template to be more like that, but I was far enough along by the time that issue was created that I decided to handle that in a follow-up PR.

Currently, I'm just brute-force overwriting whatever's in the issue body currently if we're updating the tracking issue, and I'll need to rework that to instead parse the existing body as markdown and only overwrite the parts that the bot knows/cares about while preserving the rest in a consistent structure.